runtimes/rust/src/api.rs

```diff
@@ -241,7 +241,7 @@ impl ServiceApiScope {
     }
 
     /// Return a mutable reference to the underlying web backend.
-    pub fn web_backend(&mut self) -> &mut crate::api::backends::actix::ApiBuilder {
+    pub fn web_backend(&mut self) -> &mut exonum::api::backends::actix::ApiBuilder {
```

Comment:
Seems that this method is only used in `testkit` counter service for an example authorization. Maybe remove it at all?

Reply:
It was / is used in some projects with the same purpose it's used in testkit, i.e., to implement endpoint authorization. Since there is no other ways to provide these capabilities, I'd be wary to remove this method. It may be marked as hidden / unstable, though.

Reply:
I am not sure. Without web backend you cannot write custom http handlers.